src/components/_badge.scss

```diff
@@ -0,0 +1,39 @@
+//* sass-lint:disable no-important *//
+
+// --- SUI Badge Notification --- //
+// @include sui-badge-notification();
+// @include sui-badge-notification($pseudo:true);
+// Markup ---> data-notification-counter={5}
+
+%sui-badge-notification-element {
```

Comment:
Placeholders are absolutely forbidden 😅 : https://github.com/SUI-Components/sui-theme/pull/12. They may cause code duplication in compiled files, and so that they can also overwrite other selectors that uses the same placeholder. In conclusion better convert it to a mixin 🍏

Reply:
Ok, placeholder converted to mixin. :thumbsup: